openstack/cronus/sub-charts/maillog/templates/deployment.yaml

```diff
@@ -61,11 +44,7 @@ spec:
               port: http
             initialDelaySeconds: 3
             periodSeconds: 2
-      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod | default 60 }}
-      dnsConfig:
-        options:
-        - name: ndots
-          value: "1"
+      terminationGracePeriodSeconds: 60
```

Comment:
I dont think this should come from the values as it should be immutable and not overridden in other envs